Documentation/Tutorials/Tutorial2.md

```diff
@@ -345,42 +316,35 @@ struct ReceiptElement: ProxyElement {
     let purchase = Purchase.sample
 
     var elementRepresentation: Element {
-        let column = Column { col in
-
-            col.minimumVerticalSpacing = 16.0
-            col.horizontalAlignment = .fill
-
+        Column(alignment: .fill, minimumSpacing: 16) {
             for item in purchase.items {
-                col.add(
-                    child: LineItemElement(
-                        title: item.name,
-                        price: item.price))
+                LineItemElement(
+                    title: item.name,
+                    price: item.price
+                )
             }
 
-            col.add(
-                child: LineItemElement(
-                    title: "Subtotal",
-                    price: purchase.subtotal))
-
-            col.add(
-                child: LineItemElement(
-                    title: "Tax",
-                    price: purchase.tax))
+            RuleElement()
 
-            col.add(
-                child: LineItemElement(
-                    title: "Total",
-                    price: purchase.total))
-        }
+            LineItemElement(
+                title: "Subtotal",
+                price: purchase.subtotal
+            )
 
-        let inset = Inset(
-            uniformInset: 24.0,
-            wrapping: column)
+            LineItemElement(
+                title: "Tax",
+                price: purchase.tax
+            )
 
-        var scrollView = ScrollView(wrapping: inset)
-        scrollView.contentSize = .fittingHeight
-        scrollView.alwaysBounceVertical = true
-        return scrollView
+            LineItemElement(
+                title: "Total",
+                price: purchase.total
+            )
+        }
+        .inset(uniform: 24)
+        .scrollable { scrollable in
+            scrollable.alwaysBounceVertical = true
```

Comment:
```suggestion
        .scrollable { scrollView in
            scrollView.alwaysBounceVertical = true
```

Reply:
This was the change that didn't save during our padding!

ffb74e7